src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -19,14 +19,16 @@ export default class SaveInProgressIntro extends React.Component {
         const savedAt = this.props.lastSavedDate
           ? moment(this.props.lastSavedDate)
           : moment.unix(savedForm.last_updated);
-        const expirationDate = moment.unix(savedForm.metadata.expires_at);
+        const expirationDate = moment.unix(savedForm.metadata.expires_at).format('M/D/YYYY');
 
         alert = (
           <div>
             <div className="usa-alert usa-alert-info no-background-image schemaform-sip-alert">
-              <div style={{ paddingBottom: '8px' }}>Application status: <strong>In progress</strong></div>
-              <br/>
-              <div>Last saved on {savedAt.format('MM/DD/YYYY [at] h:mm a')} <span className="schemaform-sip-expires">Your saved application will expire in {dateDiffDesc(expirationDate)}</span>.</div>
+              <div style={{ paddingBottom: '16px' }}>Application status: <strong>In progress</strong></div>
```

Comment:
Can we move this inline style into a css class?

Reply:
Great idea, this style has been moved into `schemaform-sip-alert-title`.